apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -73,12 +98,64 @@ function handleClear() {
     }
 }
 
+function ClearScheduleButton() {
+    return (
+        <Tooltip title="Clear Schedule">
+            <IconButton sx={buttonSx} onClick={handleClear}>
+                <DeleteOutline />
+            </IconButton>
+        </Tooltip>
+    );
+}
+
 function createCopyHandler(index: number) {
     return () => {
         copySchedule(index);
     };
 }
 
```

Comment:
Can you switch the order of the function declarations? I like to have helper functions in one area, and function components in another. It just helps me distinguish which parts of the file are responsible for rendering UI.

```suggestion
function createCopyHandler(index: number) {
    return () => {
        copySchedule(index);
    };
}

function ClearScheduleButton() {
    return (
        <Tooltip title="Clear Schedule">
            <IconButton sx={buttonSx} onClick={handleClear}>
                <DeleteOutline />
            </IconButton>
        </Tooltip>
    );
}
```